config/initializers/rack_attack.rb

```diff
@@ -27,6 +27,7 @@ def http_auth_token = HttpAuthenticationToken.from_header(env['HTTP_AUTHORIZATIO
   next 12 if req.post? && req.routed_to == 'api/solutions/submissions#create'
   next 30 if req.post? && req.routed_to == 'api/markdown#parse'
   next 30 if req.patch? && req.routed_to == 'api/mentoring/testimonials#reveal'
+  next 30 if req.patch? && req.routed_to == 'api/tracks/trophies#reveal'
```

Comment:
This rate limit brings it in line with the rate limit for revealing testimonials